src/blockchain/calls/instant.tsx

```diff
@@ -48,3 +49,63 @@ export const instantOrder: TransactionDef<InstantOrderData> = {
       Create Buy Order <Money value={buyAmount} token={buyToken}/>
     </>,
 };
+
+export interface GetBuyAmountData {
+  sellToken: string;
+  buyToken: string;
+  amount: BigNumber;
+}
+
+export const getBuyAmount: CallDef<GetBuyAmountData, BigNumber> = {
+  call: (_: GetBuyAmountData, context: NetworkConfig) => {
+    return context.otc.contract.getBuyAmount;
+  },
+  prepareArgs: ({ sellToken, buyToken, amount }: GetBuyAmountData, context: NetworkConfig) => [
+    context.tokens[buyToken].address,
+    context.tokens[sellToken].address,
+    amountToWei(amount, sellToken).toFixed(0)
+  ],
+  postprocess: (result, { sellToken }) => amountFromWei(result, sellToken),
```

Comment:
Here you don't have BigNumber type and in the `geyPayAmount` to you :) so which one is it ? Same goes for the token type